libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -588,6 +591,11 @@ class AP_AHRS
     }
 
 protected:
+    void update_nmea_out() {
+        if (_nmea_out != nullptr) {
```

Comment:
Why not run ::probe() in here?

Reply:
Main reason: I didn't want to always be checking it at 400Hz if the user hasn't set the parameter. It means you need to restart after setting the parameter but I think that's a reasonable compromise.